modules/service/src/main/resources/graphql/itc.graphql

```diff
@@ -3033,14 +3035,10 @@ input SpectroscopyScienceRequirementsInput {
   resolution: PosInt
 
   """
-  The signalToNoise field may be unset by assigning a null value, or ignored by skipping it altogether
-  """
-  signalToNoise: SignalToNoise
-
-  """
-  The signalToNoiseAt field may be unset by assigning a null value, or ignored by skipping it altogether
+  ExposureTimeMode, which may be unset by assigning a null value, or ignored by
+  skipping it altogether.
   """
-  signalToNoiseAt: WavelengthInput
+  exposureTimeMode: ExposureTimeModeInput
```

Comment:
I fail to find `ExposureTimeModeInput`. Should we accept both options at this level?